Sources/FoundationEssentials/TimeZone/TimeZone_Cache.swift

```diff
@@ -443,19 +410,10 @@ struct TimeZoneCache : Sendable {
     }
 
     func setDefault(_ tz: TimeZone?) {
-        let oldDefaultTimeZone = lock.withLock {
-            return $0.setDefaultTimeZone(tz)
-        }
+        lock.withLock { $0.setDefaultTimeZone(tz) }
 
-        CalendarCache.cache.reset()
-#if FOUNDATION_FRAMEWORK
-        if let oldDefaultTimeZone {
-            let noteName = CFNotificationName(rawValue: "kCFTimeZoneSystemTimeZoneDidChangeNotification-2" as CFString)
```

Comment:
Where was this moved to?

Reply:
Here: https://github.com/apple/swift-foundation/pull/719/files/dfd06a4b134fdf01ad87895e68a039847bad660c#diff-57dc8205ad14313af943e06c4553fb2e90febad0fec5a6bd8e3ffdf5e9de880eR71